### PR TITLE
HCAP-706 | Refactor server js

### PR DESCRIPTION
**Pull request by arranfw**
- [x] [Participant File](https://freshworks.atlassian.net/browse/HCAP-790)
- [x] [Employer-Form File](https://freshworks.atlassian.net/browse/HCAP-791)
- [x] [Employer-Sites file](https://freshworks.atlassian.net/browse/HCAP-792)
- [x] [User File](https://freshworks.atlassian.net/browse/HCAP-793)
- [x] [Milestone report](https://freshworks.atlassian.net/browse/HCAP-801)

### CODE REVIEW

File: docker-compose.dev.yml [hunk 1]
```diff
@@ -50,6 +50,8 @@ services:
       - CHES_HOST=${CHES_HOST}
       - AUTH_URL=${AUTH_URL}
       - CLIENT_URL=${CLIENT_URL}
+      - KC_TEST_SUPER_UER_PWD=${KC_TEST_SUPER_UER_PWD}
```
Comment (Zenen-freshworks) [inline docker-compose.dev.yml:53]: typo: SUPER_USER

File: server/routes/employer-sites.js [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+
+// Index routes
+
+// Create: Post
+router.post(
+  '/',
+  [
+    keycloak.allowRolesMiddleware('ministry_of_health'),
+    keycloak.getUserInfoMiddleware(),
+    expressRequestBodyValidator(CreateSiteSchema),
```
Comment (Zenen-freshworks) [inline server/routes/employer-sites.js:24]: This is sweet, good encapsulation

File: server/server.js [hunk 3]
```diff
@@ -99,729 +54,16 @@ app.use(
...
-      maximusId: null,
-      postalCodeFsa: req.body.postalCode.substr(0, 3),
-      preferredLocation: req.body.preferredLocation.join(';'),
-      interested: 'yes',
-      nonHCAP: null,
-      crcClear: null,
-      callbackStatus: false,
-      userUpdatedAt: new Date().toJSON(),
-    };
+app.use(`${apiBaseUrl}`, apiRouter);
```
Comment (Zenen-freshworks) [inline server/server.js:60]: Is the psiRouter going to need to be placed in the apiRouter once this feature is merged?

**Review by dbayly-freshworks — APPROVED** [review]

**Review by Zenen-freshworks — APPROVED** [review]
Nice

**Pull request merged**